packages/flutter/lib/src/widgets/radio_group.dart

```diff
@@ -0,0 +1,327 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show SemanticsRole;
+
+import 'package:collection/collection.dart';
+import 'package:flutter/services.dart';
+
+import 'actions.dart';
+import 'basic.dart';
+import 'focus_manager.dart';
+import 'focus_traversal.dart';
+import 'framework.dart';
+import 'shortcuts.dart';
+
+/// A group for radios.
+///
+/// This widget treats all radios, such as [RawRadio], [Radio], [CupertinoRadio]
+/// in the sub tree with the same type T as a group. Radios with different types
+/// are not included in the group.
+///
+/// This widget handles the group value for the radios in the subtree with the
+/// same value type.
+///
+/// Using this widget also provides keyboard navigation and semantics for the
+/// radio buttons that matches [APG](https://www.w3.org/WAI/ARIA/apg/patterns/radio/)
+///
+/// The keyboard behaviors are:
+/// * Tab and Shift+Tab: moves focus into and out of radio group. When focus
+///   moves into a radio group and a radio button is select, focus is set on
+///   selected button. Otherwise, it focus the first radio button in reading
+///   order
+/// * Space: toggle the selection on the focused radio button
```

Comment:
Missing a period at the end.

Reply:
Should enter do the same thing?

Also, maybe on the web the behavior is different? I see that the radio gets selected when I focus it. https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input/radio (probably for a different PR though)